src/pages/index.ejs

```diff
@@ -251,13 +256,16 @@ Features are implemented as preventing arming when the drone isn’t leveled and
 											<i class="fa fa-exclamation-circle fa-3x"></i>
 										</div>
 										<div class="desc">
-											<a href="#"><h4 class="text-white">BETA TESTING (WARNING)</h4></a>
+											<h4 class="text-white">BETA TESTING (WARNING)</h4>
 											<p>If you want to contribute to better development you can download the latest beta build directly from:</p>
-											<p><a href="https://ci.betaflight.tech/job/Betaflight/lastBuild/artifact/obj/">Nightly builds</a></p> 
-											</p><p>
```

Comment:
Just as information, there is removed an extra `</p>` that i think is causing the smartphone style issues